File: Entities/Paquete.cs
namespace Entidades
{
    using System;

    public class Paquete
    {
        private int varIdPaqute;
        private int varIdProveedor;
        private string varNombreProveedor;

        public int IdPaquete
        {
            get => 
                this.varIdPaqute;
            set => 
                this.varIdPaqute = value;
        }

        public int IdProveedor
        {
            get => 
                this.varIdProveedor;
            set => 
                this.varIdProveedor = value;
        }

        public string NombreProveedor
        {
            get => 
                this.varNombreProveedor;
            set => 
                this.varNombreProveedor = value;
        }
    }
}

File: Entidades/Paquete.vb
Public Class Paquete


    Private varIdPaqute As Integer
    Public Property IdPaquete() As Integer
        Get
            Return varIdPaqute
        End Get
        Set(ByVal value As Integer)
            varIdPaqute = value
        End Set
    End Property


    Private varIdProveedor As Integer
    Public Property IdProveedor() As Integer
        Get
            Return varIdProveedor
        End Get
        Set(ByVal value As Integer)
            varIdProveedor = value
        End Set
    End Property

    Private varNombreProveedor As String
    Public Property NombreProveedor() As String
        Get
            Return varNombreProveedor
        End Get
        Set(ByVal value As String)
            varNombreProveedor = value
        End Set
    End Property

End Class
